### ISSUE TITLE
Discord for vscode

### ISSUE THREAD

**Issue by tejasag** [OnlyBens/vscode-extension-ideas#9]
Maybe some sort of discord for vscode?
Well I know the voice channel function would be a problem but I think it would still be cool without it :D
Just general text options inside vscode will be a very cool thing in its own
(and who knows maybe Microsoft might buy it 😜)

Comment on this issue if you like the idea

**Comment by charfweh** [comment]
I think there's an extension that lets discord integrate with vscode.

**Comment by samarmohan** [comment]
The extension only displays what file/project you are working on in your status

**Comment by charfweh** [comment]
no, I'm talking about this https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.discord-chat
